Lib/unittest/test/test_async_case.py

```diff
@@ -170,21 +170,22 @@ async def on_cleanup(self):
     def test_deprecation_of_return_val_from_test(self):
         # Issue 41322 - deprecate return of value!=None from a test
         class Test(unittest.IsolatedAsyncioTestCase):
-            async def test(self):
+            async def test1(self):
                 return 1
             async def test2(self):
                 yield 1
 
         with self.assertWarns(DeprecationWarning) as w:
-            Test('test').run()
-            self.assertIn('It is deprecated to return a value!=None', w.warnings[0].message.args[0])
-            self.assertIn('test', w.warnings[0].message.args[0])
-            self.assertIn(w.warnings[0].filename, __file__)
+            Test('test1').run()
+        self.assertIn('It is deprecated to return a value!=None', str(w.warnings[0].message))
+        self.assertIn('test1', str(w.warnings[0].message))
+        self.assertEquals(w.warnings[0].filename, __file__)
```

Comment:
`assertEquals` is a deprecated alias. Use `assertEqual`.

Reply:
Ah, somehow missed the deprecation warnings in a PR for deprecation warnings :) thanks for catching that.